src/Engines/thinEngine.ts

```diff
@@ -649,6 +649,15 @@ export class ThinEngine {
         return ThinEngine._createCanvas(width, height);
     }
 
+    /**
+     * Create an image to use with canvas
+     * @return IImage interface
+     */
+     public createCanvasImage(): IImage

```

Comment:
Nit picking but unsure about the name, let s add @bghgary as well as he is good with naming which is weird for a dev :-)